source/Command.cpp

```diff
@@ -39,46 +39,48 @@ namespace {
 	// to display a warning to the player.
 	map<int, int> keycodeCount;
 	// Need a uint64_t 1 to generate Commands.
-	const uint64_t ONE = 1;
+	const uint64_t ONE = 1uLL;
 }
 
 
 
 // Command enumeration, including the descriptive strings that are used for the
 // commands both in the preferences panel and in the saved key settings.
-const Command Command::NONE(0, "");
+const Command Command::NONE(0uLL, "");
```

Comment:
I don't think it's strictly necessary to include the type specifier on the literals in these cases.
If it were `1 << 35`, the specifier is necessary, because it leaves ambiguity: `1` is an `int`, which is only at least 16 bits, so if it tries using `int::operator<<`, it'll do bad things because it cannot possibly shift that many bits and still be the desired number.
When it was `1uL`, a similar issue arises, because it'll try using the `long` version of the bit-shifft operator, which isn't guaranteed to be more than 32 bits.
But, with these two cases, no such operator ambiguity exists. The first one will happily cast the literal to the appropriate type (`uint64_t` in this case), same for the second. And, I think these can even be done at compile time, so it doesn't even cost any performance.

It is, of course, also not strictly necessary to drop the type specifiers, but if the goal is to minimise the impact of needing to change the type again (if we have to), I think it would make sense to just use `1` and `0` here, instead of either leaving mismatching specifiers if it is changed to something other than a `uint64_t` in the future or needing to also change the literals to match the new type, for consistency.

Reply:
I can revert the types in those two cases.

Reply:
I attributed code authorship with your suggestion in c2e4685de01ae67ea4a50494fa9c5e994b7eabdc